cmd/config.go

```diff
@@ -241,3 +254,33 @@ func printValues(values pipeline.Values) {
 		fmt.Printf("\t%s:\t%s", key, value)
 	}
 }
+
+type CollaborationResult struct {
+	VcsTye    string `json:"vcs_type"`
+	OrgSlug   string `json:"slug"`
+	OrgName   string `json:"name"`
+	OrgId     string `json:"id"`
+	AvatarUrl string `json:"avatar_url"`
+}
+
+// GetOrgCollaborations - fetches all the collaborations for a given user.
+func GetOrgCollaborations(client *rest.Client) ([]CollaborationResult, error) {
+	req, err := client.NewRequest("GET", &url.URL{Path: "me/collaborations"}, nil)
```

Comment:
nit: would be nice to make the Path a constant

Reply:
I agree, done!